ansible_collections/arista/avd/roles/eos_config_deploy_cvp/templates/cvp-devices-v3.j2

```diff
@@ -5,15 +5,17 @@
 {% else %}
 {%    set local_var.device_filter = device_filter %}
 {% endif %}
+{% set dev_cntr = namespace(value=0) %}
```

Comment:
Space around ```=``` within namespace.

Reply:
No, according to PEP-8 this is correct. Adding spaces would be wrong.